src/components/formio/textfield/textfield.component.tsx

```diff
@@ -1,39 +1,38 @@
-import {CharCount, Component, Description, Errors, Label} from '@components';
 import {IComponentProps} from '@types';
 import clsx from 'clsx';
 import {ComponentSchema} from 'formiojs';
 import React, {useState} from 'react';
 
+import {CharCount, Component, Description, Errors, Label} from '../../utils';
+
 export interface ITextFieldComponent extends ComponentSchema {
```

Comment:
can you maybe add a `// TODO: replace with @open-formulieren/open-forms-types TextFieldSchema type when its available`?